resources/assets/coffee/react/beatmapset-page/header.coffee

```diff
@@ -201,6 +189,30 @@ export class Header extends React.Component
             timeElapsed: @props.timeElapsed
 
 
+  renderAvailabilityInfo: =>
+    if currentUser.id? && @hasAvailabilityInfo()
+      href = if @props.beatmapset.availability.more_information == 'rule_violation'
+               "#{route('wiki.show', locale: currentLocale, path: 'Rules')}#beatmap-submission-rules"
```

Comment:
not quite fixable but the hash only works for english locale until #1566 is fixed :pensive:

Reply:
commonmark still doesn't support named anchors :|